lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -240,7 +241,7 @@ def call(params)
                                              create_non_existing_user_groups: create_non_existing_user_groups)
                       end
                     when 'sync_domain_client_workspaces'
-                      domain_clients = domain.clients
+                      domain_clients = domain.clients(:all, data_product)
```

Comment:
Identical code found in synchronize_user_filters.rb:157, synchronize_users.rb:244 (mass = 100)